src/manipulateToDo.js

```diff
@@ -12,6 +16,42 @@ const reorderIndices = (tasks) => {
   });
 };
 
+const drag = (ev) => {
+  const draggedElement = ev.target;
+  dragStore = draggedElement;
+  dragStore.style.backgroundColor = '#4c3c3c';
+};
+
+const dragStart = (e) => {
+  let elUnder = e.target;
+  const classArray = elUnder.classList.value.split(' ');
+  if (
+    classArray.includes('fa-check')
+    || classArray.includes('checkbox')
+    || classArray.includes('to-do')
+  ) {
+    elUnder = elUnder.parentElement.parentElement;
+  } else if (
+    classArray.includes('fa-arrows-alt')
+    || classArray.includes('two')
+  ) {
+    elUnder = elUnder.parentElement;
+  }
```

Comment:
This same code is repeating through this file several times. Making it a separate function would improve readability and stick a bit better to the YAGNI, what do you think? 🤓